### PR TITLE
fix(postprocess): Add function to calculate EN 17037 metrics

### PR DESCRIPTION
**Pull request by mikkelkp**
Needs this first: https://github.com/ladybug-tools/honeybee-radiance/pull/742

### CODE REVIEW

File: pollination/honeybee_radiance/post_process.py [hunk 1]
```diff
@@ -311,6 +311,49 @@ def calculate_annual_metrics(self):
...
+    schedule = Inputs.file(
+        description='Path to an annual schedule file. Values should be 0-1 separated '
+        'by new line. This should be a daylight hours schedule according to EN 17037.',
+        path='schedule.txt'
+    )
+
+    @command
+    def calculate_annual_metrics_en17037(self):
+        return 'honeybee-radiance post-process annual-daylight-en17037 raw_results ' \
+            'schedule.txt --sub_folder ../metrics'
```
Comment (mostaphaRoudsari) [inline pollination/honeybee_radiance/post_process.py:334]: We should keep the output folder in the same layer in Function. You can always copy it to a top level folder in the recipe.

```suggestion
            'schedule.txt --sub_folder metrics'
```

File: pollination/honeybee_radiance/post_process.py [hunk 2]
```diff
@@ -311,6 +311,49 @@ def calculate_annual_metrics(self):
...
+
+    # outputs
+    annual_en17037_metrics = Outputs.folder(
+        description='Annual metrics folder. This folder includes all the other '
+        'subfolders which are also exposed as separate outputs.', path='metrics'
```
Comment (mostaphaRoudsari) [inline pollination/honeybee_radiance/post_process.py:339]: Based on this line, I _feel_ the folder above in the command is actually a mistake.

**Review by mostaphaRoudsari — COMMENTED** [review]
Hi @mikkelkp, see my comments!

Don't we also need to update the version for honeybee-radiance dependency?

**Comment by mikkelkp** [comment]
Hi @mostaphaRoudsari, I updated the honeybee-radiance dependency. I think it that's why it says there is a conflict now? At least on my end...

For the folder you mentioned above, I used the same structure as the annual daylight metrics: https://github.com/pollination/honeybee-radiance/blob/master/pollination/honeybee_radiance/post_process.py#L277

**Comment by mostaphaRoudsari** [comment]
Hi @mikkelkp, Thanks! I'm merging this in so you can test it yourself.

**Pull request merged**